### PR TITLE
SetLabels from file patterns

### PR DESCRIPTION
**Pull request by tylerganter**
Easy way to generate `VideoSetLabels` or `ImageSetLabels` from a file pattern

### CODE REVIEW

File: eta/core/video.py [hunk 1]
```diff
@@ -1382,6 +1382,30 @@ def _apply_schema(self):
...
+        Returns:
+            a VideoSetLabels instance
+
+        '''
+        labels_list = [
+            VideoLabels.from_json(labels_path)
+            for labels_path in etau.get_pattern_matches(video_labels_patt)
+        ]
+
+        video_set_labels = VideoSetLabels()
```
Comment (benjaminpkane) [inline eta/core/video.py]: Can you use `cls()` here? For `BigVideoSetLabel`'s sake. My comment applies to `from_image_labels_patt()`, as well.

File: eta/core/image.py [hunk 2]
```diff
@@ -650,6 +650,30 @@ def _validate_schema(self):
...
+
+        Args:
+             image_labels_patt: a pattern with one or more numeric sequences:
+                example: "/path/to/labels/%05d.json"
+
+        Returns:
+            a ImageSetLabels instance
+
+        '''
+        labels_list = [
```
Comment (brimoor) [inline eta/core/image.py]: why two separate loops? seems simpler to do:
```py
image_set_labels = ImageSetLabels()
for labels_path in etau.get_pattern_matches(image_labels_patt):
    image_set_labels.add(ImageLabels.from_json(labels_path))
```

**Comment by tylerganter** [comment]
@benjaminpkane @brimoor good points. just fixed 'em

File: eta/core/image.py [hunk 3]
```diff
@@ -650,6 +650,23 @@ def _validate_schema(self):
...
+    def from_image_labels_patt(cls, image_labels_patt):
+        '''Creates an instance of `cls` from a pattern of `_ELE_CLS` files.
+
+        Args:
+             image_labels_patt: a pattern with one or more numeric sequences:
+                example: "/path/to/labels/%05d.json"
+
+        Returns:
+            a `cls` instance
+
```
Comment (brimoor) [inline eta/core/image.py]: nit: extra blank line here

**Pull request merged**